appveyor.yml

```diff
@@ -0,0 +1,47 @@
+version: "master-{build}"
+
+os: Visual Studio 2015
+platform:
+  - x64
+
+environment:
+  matrix:
+    - ruby_version: "24-x64"
+    - ruby_version: "24"
+
+clone_folder: c:\projects\win32-certstore
+clone_depth: 1
+skip_tags: true
+branches:
+  only:
+    - master
+    - chef-12
+
+install:
+  - systeminfo
+  - winrm quickconfig -q
+  - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
+  - echo %PATH%
+  - ps: $env:OMNIBUS_RUBYGEMS=$(findstr rubygems omnibus_overrides.rb | %{ $_.split(" ")[3] })
+  - echo %OMNIBUS_RUBYGEMS%
+  - ps: $env:OMNIBUS_BUNDLER=$(findstr bundler omnibus_overrides.rb | %{ $_.split(" ")[3] })
+  - echo %OMNIBUS_BUNDLER%
+  - ruby --version
+  - gem --version
+  - bundler --version
+  - gem uninstall bundler -a -x
```

Comment:
I'm looking into some failures in chef/chef on appveyor that make me suspect we need to not try to uninstall bundler. See https://github.com/chef/chef/pull/6597.

Reply:
Thanks @btm, I will remove this line.